Python/susankoju/task3/read_sqlite.py

```diff
@@ -0,0 +1,8 @@
+import sqlite3
+
+conn = sqlite3.connect('output/results.db')
+with conn:
+    for row in conn.execute('SELECT * FROM data'):
+        print(row)
+
+conn.close()
```

Comment:
No need to close the connection. 
The context manager 'with' will handle the connection closing operation.